tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/matterdashboard.controller.js

```diff
@@ -1809,29 +1810,30 @@
             //#region Exporting to Excel Test
             vm.export = function () {
                 //vm.lazyloaderdashboard = false;
-                if (vm.tabClicked != "Pinned Matters") {
-                    var exportMatterSearchRequest = {
-                        Client: {
-                            Url: configs.global.repositoryUrl
-                        },
-                        SearchObject: {
-                            PageNumber: 1,
-                            ItemsPerPage: 500,
-                            SearchTerm: jsonMatterSearchRequest.SearchObject.SearchTerm,
-                            Filters: {
-                                ClientsList: jsonMatterSearchRequest.SearchObject.Filters.ClientsList,
-                                PGList: jsonMatterSearchRequest.SearchObject.Filters.PGList,
-                                AOLList: jsonMatterSearchRequest.SearchObject.Filters.AOLList,
-                                FromDate: jsonMatterSearchRequest.SearchObject.Filters.FromDate,
-                                ToDate: jsonMatterSearchRequest.SearchObject.Filters.ToDate,
-                                FilterByMe: jsonMatterSearchRequest.SearchObject.Filters.FilterByMe
-                            },
-                            Sort: {
-                                ByProperty: jsonMatterSearchRequest.SearchObject.Sort.ByProperty,
-                                Direction: jsonMatterSearchRequest.SearchObject.Sort.Direction
-                            }
+                var exportMatterSearchRequest = {
+                    Client: {
+                        Url: configs.global.repositoryUrl
+                    },
+                    SearchObject: {
+                        PageNumber: 1,
+                        ItemsPerPage: 500,
+                        SearchTerm: jsonMatterSearchRequest.SearchObject.SearchTerm,
+                        Filters: {
+                            ClientsList: jsonMatterSearchRequest.SearchObject.Filters.ClientsList,
+                            PGList: jsonMatterSearchRequest.SearchObject.Filters.PGList,
+                            AOLList: jsonMatterSearchRequest.SearchObject.Filters.AOLList,
+                            FromDate: jsonMatterSearchRequest.SearchObject.Filters.FromDate,
+                            ToDate: jsonMatterSearchRequest.SearchObject.Filters.ToDate,
+                            FilterByMe: jsonMatterSearchRequest.SearchObject.Filters.FilterByMe
+                        },
+                        Sort: {
+                            ByProperty: jsonMatterSearchRequest.SearchObject.Sort.ByProperty,
+                            Direction: jsonMatterSearchRequest.SearchObject.Sort.Direction
                         }
-                    };
+                    }
+                };
+                if (vm.tabClicked != "Pinned Matters") {

```

Comment:
@kiran11990  should not compare with hard coded text. get this value from config.